sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_client.py

```diff
@@ -117,24 +150,42 @@ def get_job_status(self, job_id, **kwargs):
     @distributed_trace
     def cancel_job(self, job_id, **kwargs):
         # type: (str, **Any) -> None
-        """
+        """Cancel a currently processing or queued job.
+
+        A job will not be cancelled if it is already completed, failed, or cancelling.
+        All documents that have completed translation will not be cancelled and will be charged.
+        All pending documents will be cancelled if possible.
```

Comment:
also, should we use `canceled` instead of `cancelled` when it is in docs and not referring to the `status` variable?

Reply:
Since either spelling works, I kind of feel the opposite - like we should be consistent across the whole library. It feels weird to switch back and forth in the same sentence like

`:ivar int documents_cancelled_count: Number of documents that were canceled for translation.`

Reply:
Fair. I need to check what we do in .NET :)